packages/osd-std/src/validate_object.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
If this is a new file, then the new license header is all that is needed like this: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/dev-tools/get-version.sh#L3

Reply:
I updated license header.